brontide/conn.go

```diff
@@ -32,9 +32,19 @@ var _ net.Conn = (*Conn)(nil)
 // remote peer located at address which has remotePub as its long-term static
 // public key. In the case of a handshake failure, the connection is closed and
 // a non-nil error is returned.
-func Dial(localPriv *btcec.PrivateKey, netAddr *lnwire.NetAddress) (*Conn, error) {
+func Dial(localPriv *btcec.PrivateKey, netAddr *lnwire.NetAddress,
```

Comment:
I like the second option - I wanted to change the method signature and still allow other packages to call `Dial` in the same way that they were. I prefer a config that defaults to `net.Dial` and maintains the method signature to the current implementation. Maybe @roasbeef can chime in since he suggested the vararg?